src/context.ts

```diff
@@ -75,3 +77,60 @@ export class OperonContextImpl implements OperonContext {
     return value as T;
   }
 }
+
+
+/**
+ * TODO : move logger and application, getConfig to a BaseContext which is at the root of all contexts
+ */
+export class InitContext {
+  
+  readonly logger: Logger ;
+
+  // All private Not exposed
+  private userDatabase: UserDatabase;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private application: any;
+
+  constructor(readonly operon: Operon) {
+    this.logger = operon.logger;
+    this.userDatabase = operon.userDatabase;
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    this.application = operon.config.application;
+
+  }
+
+  createUserSchema(): Promise<void> {
+    return this.userDatabase.createSchema()  ;
+  }
+    
+  dropUserSchema(): Promise<void> {
+    return this.userDatabase.dropSchema();
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  queryUserDB<R>(sql: string, ...params: any[]): Promise<R[]> {
```

Comment:
This should be unknown[], which is more appropriate for the situation and also removes the need for the lint directive.